src/main/java/meteordevelopment/meteorclient/systems/modules/player/AutoTool.java

```diff
@@ -260,6 +264,9 @@ && isFortunable(state.getBlock())
         if (itemStack.isIn(ItemTags.SWORDS) && (state.getBlock() instanceof BambooBlock || state.getBlock() instanceof BambooShootBlock))
             score += 9000 + (itemStack.get(DataComponentTypes.TOOL).getSpeed(state) * 1000);
 
+        // Prefer hoes slightly for crops when fortune is enabled and the block is a crop.
+        if (fortuneOre && (state.getBlock() instanceof CropBlock || state.getBlock() instanceof NetherWartBlock) && itemStack.isIn(ItemTags.HOES)) score += 10;
```

Comment:
there's no point to prefer hoes as default when we don't consider using fortune for crops, thats why its req fortuneore option